lib/factorial.rb

```diff
@@ -0,0 +1,5 @@
+# Computes factorial of the input number and returns it
+def factorial(number)
+  puts "NOT IMPLEMENTED"
+  return number
```

Comment:
How would you feel about doing `raise NotImplementedError` here instead?

Best practice I think is generally to avoid `puts` statements and other things that will print to the console for code being tested, because that output kind of gets randomly interspersed with the test runner's output.

Reply:
Done!